.pre-commit-config.yaml

```diff
@@ -0,0 +1,37 @@
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 3.9.2
+    hooks:
+      - id: flake8
+        additional_dependencies: ["flake8-eradicate==0.4.0"]
+
+  - repo: https://github.com/patrick91/pre-commit-alex
+    rev: aa5da9e54b92ab7284feddeaf52edf14b1690de3
+    hooks:
+      - id: alex
+        exclude: CHANGELOG.md
+
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.3.2
+    hooks:
+      - id: prettier
+        files: '^docs/.*\.mdx?$'
```

Comment:
shall we do this for all md files?

Reply:
Oh yeah we should. I just copied this from the main repo!